src/components/Web3Status/index.tsx

```diff
@@ -195,11 +251,23 @@ function Web3StatusInner() {
         properties={{ received_swap_quote: validSwapQuote }}
         element={ElementName.CONNECT_WALLET_BUTTON}
       >
-        <Web3StatusConnect onClick={toggleWallet} faded={!account}>
-          <Text>
-            <Trans>Connect Wallet</Trans>
-          </Text>
-        </Web3StatusConnect>
+        {navbarFlag === NavBarVariant.Enabled ? (
+          <Web3StatusConnectNavbar faded={!account}>
```

Comment:
perhaps could make this flex with align items center so you don't need the margin top/bottom auto? up to you